app/bingey-ui/src/components/WatchlistsControler/WatchlistPicker/WatchlistPicker.scss

```diff
@@ -1,3 +1,7 @@
+.center {
```

Comment:
Watchlist picker was using this class from title-card stylesheet that was removed